Orange/widgets/visualize/tests/test_owpythagorastree.py

```diff
@@ -1,12 +1,22 @@
 """Tests for the Pythagorean tree widget and associated classes."""
 import math
 import unittest
+from unittest.mock import Mock
 
+from Orange.classification import TreeLearner
+from Orange.data import Table
+from Orange.regression.tree import TreeRegressionLearner
```

Comment:
After rebasing to master, you'll need to change this to
`from Orange.regression.tree import TreeLearner as TreeRegressionLearner`